playground/async-chunk-css-order/__tests__/async-chunk-css-order.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, test } from 'vitest'
+import { getColor, isBuild, page } from '~utils'
+
+describe.runIf(isBuild)('build', () => {
+  test('should apply correct style', async () => {
```

Comment:
Should this maybe go in the `css` playground, as a subfolder, instead of its own playground?

Reply:
Ah yes I didn't notice this is using React.  The tests in @poyoho's PRs didn't use react, I think this could be refactored to remove React in a similar way.

Reply:
> The ideal would be if this could be added directly to the `css` playground without React. But if not, we may want to give this new playground a more generic name so we can test other things on it (it is similar to the `preload` that uses Vue also)

Need i update the test file in the pr and remove react?

Reply:
@sanyuan0704 maybe you could remove the new playground and use the one from @poyoho testing this particular bug here https://github.com/vitejs/vite/pull/9278. They are already coded for the css playground

Reply:
Okay, get it.

Reply:
i have optimized the test cases and remove react in them.